src/utils/responseUtil.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Utility class to build the response object
+ */
+class ResponseUtil {
+   /**
+   * builds successful response object
+   * @param {Object} data data object to process
+   * @return response
+   */
+  buildSuccessfulResponse (data) {
```

Comment:
What about a default success status of 200, with optional param for status_code? just thinking on the long run. Could be useful for later, if we implement updates on mail setting through the api.

Reply:
Can't think of a use case where we will need to update email? Can you give me an example?

Reply:
If we decide to have an endpoint that creates a new mail settings:
- email: jamfree@gmail.com
- password: 1234wtv

Which would be used to send mails. in that case you would return a `201`. The flexibility would be nice its not over complicating anything.

Reply:
I thought we were going to forget about this feature since we thought it was overkill? I do agree that it looks cleaner with a default status code though so I can add it. But as for the add email use case, let me know if we're still planning to do this?

Reply:
Added default `statusCode`